test/jasmine/tests/plot_interact_test.js

```diff
@@ -12,7 +12,7 @@ describe('Test plot structure', function () {
 
     afterEach(destroyGraphDiv);
 
-    describe('cartesian plots', function() {
+    describe('cartesian plots', function() {        
```

Comment:
@cldougl you might want to try http://nategood.com/sublime-text-strip-whitespace-save

to avoid those nasty diff lines.